osquery/tables/system/posix/ulimit_info.cpp

```diff
@@ -0,0 +1,96 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#include <cerrno>
+#include <climits>
+#include <map>
+#include <string>
+
+#include <sys/resource.h>
+
+#include <osquery/logger.h>
+#include <osquery/tables.h>
+
+namespace osquery {
+namespace tables {
+
+void getLimit(QueryData& results) {
+  static const std::map<std::string, int> resource_map = {
```

Comment:
Move this out of function scope and name it kLimitsResourceMap or something similar for consistency with other constants.